drivers/flash/nrf_qspi_nor.c

```diff
@@ -1294,7 +1318,7 @@ static int exit_dpd(const struct device *const dev)
 {
 	if (IS_ENABLED(DT_INST_PROP(0, has_dpd))) {
 		struct qspi_cmd cmd = {
-			.op_code = SPI_NOR_CMD_RDPD,
+		       .op_code = SPI_NOR_CMD_RDPD,
```

Comment:
Please remove this change as well.

Reply:
Bad editor automatically adding spaces. Cleaned  up.